lib/commons/aria/index.js

```diff
@@ -55,21 +59,24 @@ lookupTable.attributes = {
 		values: ['copy', 'move', 'reference', 'execute', 'popup', 'none']
 	},
 	'aria-errormessage': {
-		type: 'idref'
+		type: 'idref',
+		allowEmpty: true
```

Comment:
Is this new? If so, where are the tests for it?

Reply:
In order to make this configurable per attribute I added this. This property is used in `aria.validateAttrValue`, which is where the tests for it are. All changes (see PR description) are also tested in integration tests.

Reply:
There is no current test to test that this passes when empty